application/views/PaymentSuccessful.php

```diff
@@ -20,9 +20,15 @@
     </head>
 
     <body>
+        
+
         <div class="centre-page">
+            <!--
+            The href of this button link is left empty as it would otherwise not work on localhost.
+            When the page is finished, this should be changed to the websites url "aspa.wdcc.co.nz"
+            -->
+            <br><a href="/"><img align="left" alt="Home Page Button" alt="Home Page" src="assets/images/iconmonstr-home-grey.svg" width="30px"></a><br><br>
```

Comment:
1) Can you close the `<br />` statements and put these on separate lines for greater code visibility? Also, close the `img` tag too.

2) Let's also separate out the `<a>` tag and the `<img>` tag on separate lines and indented, like so...
```
<br />
<a ...>
    <img ... />
</a>
<br />
```